install/updateVersion.sh

```diff
@@ -53,6 +54,7 @@ while getopts :gi:n:p:x:c:sA:P:E: arg; do
     p) PILOT_HUB_TAG="${OPTARG}";; # Format: "<hub>,<tag>"
     x) MIXER_HUB_TAG="${OPTARG}";; # Format: "<hub>,<tag>"
     c) CA_HUB_TAG="${OPTARG}";; # Format: "<hub>,<tag>"
+    r) PROXY_TAG="${OPTARG}";;
```

Comment:
you need to add r in the getopts as well

Reply:
Good catch